benchmarks/ClusterBenchmark/Program.cs

```diff
@@ -228,14 +234,16 @@ async Task RunBatch(Random? rnd, Cluster cluster)
             {
                 var requests = new List<Task>();
 
+                
                 try
                 {
                     var ct = CancellationTokens.FromSeconds(20);
 
+                    var ctx = cluster.System.Root.Batch(batchSize,ct);
                     for (var i = 0; i < batchSize; i++)
                     {
                         var id = "myactor" + rnd.Next(0, actorCount);
-                        var request = SendRequest(cluster, id, ct);
+                        var request = SendRequest(cluster, id, CancellationToken.None , ctx);
```

Comment:
unsure on what this is needed for if we use a batch context that already have a ct?

Reply:
The batch future should actually check if it is the same token as set on the parent batch process. So theoretically the same token passed to each future should not affect the outcome.